fhir-server-test/src/test/java/com/ibm/fhir/server/test/ImportOperationTest.java

```diff
@@ -0,0 +1,246 @@
+        Parameter part2 = Parameter.builder().name(string("url")).value(Url.of(url)).build();
+
+        // Required: Input (relative to bucket)
+        Parameter inputParameter = Parameter.builder().name(string("input")).part(part1, part2).build();
+        parameters.add(inputParameter);
+
+        // Optional: Storage Detail
+        Parameter storageDetailParameter =
+                Parameter.builder().name(string("storageDetail")).value(string("ibm-cos")).build();
+        parameters.add(storageDetailParameter);
+
+        Parameters.Builder builder = Parameters.builder();
+        builder.id(UUID.randomUUID().toString());
+        builder.parameter(parameters);
+        Parameters ps = builder.build();
+
+        try (StringWriter writer = new StringWriter();) {
+            FHIRGenerator.generator(Format.JSON, true).generate(ps, writer);
+            if (DEBUG) {
+                System.out.println(writer.toString());
+            }
+        }
+        return ps;
+    }
+
+    /**
+     * add query parameter list
+     */
+    public WebTarget addQueryParameterList(WebTarget target, String header, List<String> vals) {
+        if (header != null && vals != null && !vals.isEmpty()) {
+            target = target.queryParam(header, vals.stream().collect(Collectors.joining(",")));
+        }
+        return target;
+    }
+
+    /**
+     * adds the query parameter
+     */
+    public WebTarget addQueryParameter(WebTarget target, String header, String val) {
+        if (header != null && val != null) {
+            target = target.queryParam(header, val);
+        }
+        return target;
+    }
+
+    public Response doGet(String path, String mimeType) {
+        WebTarget target = getWebTarget();
+        target = target.path(path);
+        return target.request(mimeType).get(Response.class);
+    }
+
+    public Response doPost(String path, String inputFormat, String inputSource, String resourceType, String url)
+            throws FHIRGeneratorException, IOException {
+        WebTarget target = getWebTarget();
+        target = target.path(path);
+        if (DEBUG) {
+            System.out.println("URL -> " + target.getUri());
+        }
+        Parameters parameters = generateParameters(inputFormat, inputSource, resourceType, url);
+        Entity<Parameters> entity = Entity.entity(parameters, FHIRMediaType.APPLICATION_FHIR_JSON);
+        return target.request(FHIRMediaType.APPLICATION_FHIR_JSON).post(entity, Response.class);
+    }
+
+    public Response polling(String statusUrl) throws InterruptedException {
+        int status = 202;
+        int totalTime = 0;
+        Response response = null;
+        while (Response.Status.ACCEPTED.getStatusCode() == status) {
+            response = doGet(statusUrl, FHIRMediaType.APPLICATION_FHIR_JSON);
+            // 202 accept means the request is still under processing
+            // 200 mean export is finished
+            status   = response.getStatus();
```

Comment:
space

Reply:
fixed, I was playing with save and alignment